CIScripts/Test/Tests/Protocol/SingleHostWithUtils.Tests.ps1

```diff
@@ -151,6 +154,8 @@ Describe "Single compute node protocol tests with utils" {
             $ContrailNM.RemoveNetwork($ContrailNetwork)
             Remove-Variable "ContrailNetwork"
         }
+
+        Merge-Logs -LogSources @($DriverLogs, $AgentLogs)
```

Comment:
What's going to happen when some earlier step in this block fails? Can we manage to handle merging logs anyway?